### PR TITLE
FIX: test pin to reports_bug branch

### PR DESCRIPTION
**Pull request by jdkent**

### CODE REVIEW

**Comment by arokem** [comment]
Hey @jdkent : thanks for doing this. Related to https://github.com/poldracklab/niworkflows/pull/443?

**Comment by jdkent** [comment]
@arokem yes! I couldn't find the exact context for @oesteban's [commit](https://github.com/josephmje/dmriprep-1/commit/b7181427cda01f1c800afd6c59aa5ce0ebd89d5d), so I'm isolating the change here.

**Comment by jdkent** [comment]
[yay, it works now](https://16-234180549-gh.circle-artifacts.com/0/tmp/THP002/derivatives/dmriprep/sub-THP0005.html)!

**Comment by oesteban** [comment]
Okay, I'm going to merge niworkflow's PR and release, so that we can add this into master (that is the cause of the failed test - not being able to have link dependencies in pypi).

File: setup.cfg [hunk 1]
```diff
@@ -24,7 +24,7 @@ install_requires =
     indexed_gzip >=0.8.8
     nibabel ~= 3.0.0rc1
     nipype ~= 1.3.1
-    niworkflows ~= 1.1.0
+    niworkflows @ git+https://github.com/jdkent/niworkflows.git@f6912851795e116771d1f82cd79999a3f32b81e3
```
Comment (oesteban) [inline setup.cfg]:
```suggestion
    niworkflows ~= 1.1.3
```

**Comment by jdkent** [comment]
thanks for the release @oesteban, updated the pin and put myself in the zenodo file :+1:

**Pull request merged**